packages/primitives/src/Image/Image.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled'
+import {
+  variant,
+  VariantProps,
+  sx,
+  shouldForwardProp,
+  systemProps,
+  SystemProps,
+} from '../Box/Box'
+
+export type ImageProps = React.DetailedHTMLProps<
+  React.HTMLAttributes<HTMLImageElement>,
+  HTMLImageElement
+> &
+  VariantProps &
+  SystemProps
+
+export const Image = styled('img', {
+  shouldForwardProp,
+})<ImageProps>(
+  {
+    boxSizing: 'border-box',
+    margin: 0,
+    minWidth: 0,
+  },
+  ({ variant: variantProp, variantKey = 'images', theme }) =>
+    variant({ variant: variantProp, variantKey, theme }),
+  ...systemProps,
```

Comment:
@seri we can get rid of `systemProps` we aren't adding them to any new components at this point and only using `sx`.